css/main.css

```diff
@@ -210,6 +250,35 @@ input:focus:-ms-input-placeholder { color:transparent;  } /* IE 10+ */
   height: 850px;
 }
 
+@media (max-width: 1100px) {
+  .nav-option-bar {
+    background: #F8AC45;
+    padding: 0;
+    position: fixed;
+    right: -400px;
+    width: 400px;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
```

Comment:
let's keep all webkit rules grouped together like this at the top or bottom of a selector section